doc/translations.md

```diff
@@ -70,6 +70,12 @@ If you only want to translate friendica into another language you wont need any
 
 For further information see the utils/README file.
 
+Help, Tobias asked me to regenerate the master messages.po file!
+----------------------------------------------------------------
+
+Don't panic.
+Just run `bin/run_xgettext.sh` from your base Friendica directory and commit `util/messages.po` to your branch. 
```

Comment:
What if he asked that for an addon? ;-)

Reply:
Ow!

Reply:
Interestingly, not even `utils/README` mention this case.

Reply:
Go into the directory of the addon you want to extract the strings from and then use the `-s` parameter of the `run_xgettext.sh` tool.

I have never used the other mode of the shell script (`-a`) to be honest.

Reply:
So 
```
cd addon/twitter
../../bin/run_xgettext.sh -s
```
?